### ISSUE TITLE
[Unity 5.6] Build error: An assembly with the same name `XInputDotNetPure' has already been imported.

### ISSUE THREAD

**Issue by pedro15** [JISyed/Unity-XboxCtrlrInput#26]
"error CS1704: An assembly with the same name `XInputDotNetPure' has already been imported. Consider removing one of the references or sign the assembly
Assets/Plugins/x86/XInputDotNetPure.dll (Location of the symbol related to previous error)
Assets/Plugins/x86_64/XInputDotNetPure.dll (Location of the symbol related to previous error)

Compilation failed: 1 error(s), 0 warnings"

So i can't compile on unity 5.6 .. :/ ( i'm on windows , target plataform: Standalone windows )

**Comment by jinnindo** [comment]
Since updating to Unity 5.6, I am also getting this error every time I try to build a standalone.

error CS1704: An assembly with the same name `XInputDotNetPure' has already been imported. Consider removing one of the references or sign the assembly
Assets/Plugins/x86/XInputDotNetPure.dll (Location of the symbol related to previous error)
Assets/Plugins/x86_64/XInputDotNetPure.dll (Location of the symbol related to previous error)

**Comment by pedro15** [comment]
It seems to be a error with the Plugin XInputDotNetPure. .. if i delete the XInputDotNetPure.dll from x86 folder and move the XInputDotNetPure.dll from x86_64 directly to Plugins it compiles ...

**Comment by iamkraya** [comment]
Filed a bug report with Unity as it appears to be on their end. Plugin Importer not respecting architecture.

**Comment by JISyed** [comment]
I recently [found out from the XInput project](https://github.com/speps/XInputDotNet/issues/17) that managed plugins in Unity (the ones written in C#, not C++) need to be built with the same architecture as the Unity Editor (32-bit or 64-bit). 

The managed plugin XInputDotNetPure.dll had both a 32-bit and 64-bit version. I had to make an executive decision and remove the 32-bit plugin because I only use 64-bit version of the Unity Editor. That should hopefully resolve the build errors, if you use 64-bit version of Unity. XboxCtrlrInput will no longer support 32-bit Unity Editor (you still should be able to build 32-bit games though).

I made [a new release](https://github.com/JISyed/Unity-XboxCtrlrInput/releases/tag/1.7.0) addressing this problem. Hopefully it has been resolved.
